cmd/libsnap-confine-private/mount-opt.c

```diff
@@ -115,3 +118,134 @@ const char *sc_mount_opt2str(unsigned long flags)
 	}
 	return buf;
 }
+
+char *sc_mount_cmd(const char *source, const char *target,
+		   const char *filesystemtype, unsigned long mountflags,
+		   const void *data)
+{
+	// NOTE: this uses static buffer because it has lower complexity than a
+	// dynamically allocated buffer. We've decided as a team to prefer this
+	// approach.
+	static char buf[PATH_MAX * 2 + 1000];
```

Comment:
"two paths + some extra space". Believe me I much preferred the version that had a dynamically allocated buffer.